cloudslang-cli/src/main/java/io/cloudslang/lang/cli/SlangCLI.java

```diff
@@ -127,11 +130,32 @@ private boolean invalidVerboseInput(String verbose) {
         return !Arrays.asList(validArguments).contains(verbose.toLowerCase());
     }
 
-    @CliCommand(value = "inspect", help = "Display metadata about an executable")
+    @CliCommand(
+            value = "inspect",
+            help = "Display metadata about an executable / Load system properties from a properties file"
+    )
     public String inspectExecutable(
-            @CliOption(key = {"", "f", "file"}, mandatory = true, help = PATH_TO_FILENAME_HELP) final File executableFile
+            @CliOption(key = {"", "f", "file"}, mandatory = true, help = PATH_TO_FILENAME_HELP) final String slangFile
     ) throws IOException {
-        return metadataHelper.extractMetadata(executableFile);
+        if (compilerHelper.isExecutable(slangFile)) {
+            return metadataHelper.extractMetadata(new File(slangFile));
+        } else {
+            Set<SystemProperty> systemProperties = compilerHelper.loadSystemProperties(Lists.newArrayList(slangFile));
```

Comment:
I think we can go with one file for now, after syncing with @SamMarkowitz , we agreed to add a separate `list properties.sl` cmd for displaying SPs from a file